frontend/packages/console-app/src/components/pdb/AvailabilityRequirementPopover.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Text, Title } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { ExternalLink, openshiftHelpBase } from '@console/internal/components/utils';
+import { FieldLevelHelp } from '@console/internal/components/utils/field-level-help';
+
+import './form-popover.scss';
+
+const AvailabilityRequirementPopover: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <FieldLevelHelp>
+      <Title headingLevel="h3">{t('console-app~Availability requirement')}</Title>
```

Comment:
```suggestion
  <Stack hasGutter>
    <StackItem>
      <Title headingLevel="h3">{t('console-app~Availability requirement')}</Title>
    </StackItem>
    <StackItem>...</StackItem>
    ... // all the way down this component; each Title/Text should get its own StackItem
```
By using `Stack`/`StackItem` from PF, you give PF the full control over the spacing. If PF wants to change things, we don't need to worry about updating our end of the agreement.

Always favour PF styling over writing your own CSS.